tools/github_update.py

```diff
@@ -96,118 +105,237 @@ def _get_repo_path(self):
         return re_match.group(1)
 
 
-    def _get_details_link_url(self, details_url):
-        '''returns the url to be included as the details link in the status'''
-        if not details_url:
-            details_url = os.environ.get('GITHUB_COMMIT_STATUS_URL', '') # custom URL via env
-        if not details_url:
-            details_url = os.environ.get('BUILD_URL', '') # provided by jenkins
-            if details_url:
-                details_url += 'console'
-        if not details_url:
-            raise Exception(
-                'Failed to determine URL for details link. ' +
-                'Provide either GITHUB_COMMIT_STATUS_URL or BUILD_URL in env.')
-        return details_url
-
-
-    def _get_auth_token(self):
+    def github_auth_token(self):
         github_token = os.environ.get('GITHUB_TOKEN_REPO_STATUS', '')
```

Comment:
Yea, and if it's set to an empty string, the following conditional will execute as true.  The above one liner has the same effect.